wgpu-native/src/command/compute.rs

```diff
@@ -66,7 +66,11 @@ pub extern "C" fn wgpu_compute_pass_set_bind_group(
     let mut pass_guard = HUB.compute_passes.write();
     let pass = &mut pass_guard[pass_id];
     let bind_group_guard = HUB.bind_groups.read();
-    let bind_group = &bind_group_guard[bind_group_id];
+
+    let bind_group = pass
+        .trackers
+        .bind_groups
+        .get_with_extended_usage(&*bind_group_guard, bind_group_id, DummyUsage).unwrap();
```

Comment:
actually, I think this is done in the `insert_barriers` below, so this particular change shouldn't be needed